fvt/predictor/predictor_suite_test.go

```diff
@@ -60,10 +62,14 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	// runs *only* on process #1
 	createFVTClient()
 
-	// confirm 3 serving runtimes exist
+	// confirm 3 serving runtimes exist in namespace scope mode
 	list, err := FVTClientInstance.ListServingRuntimes(metav1.ListOptions{})
 	Expect(err).ToNot(HaveOccurred())
-	Expect(list.Items).To(HaveLen(3))
+	if NameSpaceScopeMode {
+		Expect(list.Items).To(HaveLen(3))
+	} else {
+		Expect(list.Items).To(HaveLen(0))
+	}
```

Comment:
For these, wouldn't it be better to list either SRs or CSRs depending on namespace/cluster mode? I.e. the list should still contain 3 in either case.

Reply:
As mentioned in commit comments, I try to minimize the FVT changes to  test CSRs, for which I suggest to create a new PR.